Project.toml

```
@@ -25,16 +29,20 @@ BenchmarkTools = "^0.4.2, 0.5"
 ColorSchemes = "^3.9"
 DataFrames = "^0.21"
 GitHub = "^5.0.2"
+JLD2 = "0.1.12"
 JSON = "0.20.0, 0.21.0"
 LaTeXTabulars = "^0.1.0"
+NLPModels = "0.13"
 PkgBenchmark = "^0.2.0"
 Plots = "1.0, 1.1"
 PrettyTables = "0.9.0"
-julia = "^1.0.0"
+SolverTools = "0.2"
```

```suggestion
SolverTools = "0.3"
```
Do we need SolverTools now that most code is moved?

---

We do, for the logging.